plugins/modules/udm_dns_record.py

```diff
@@ -159,10 +147,10 @@ def main():
 
     workname = name
     if type == 'ptr_record':
-        if not HAVE_IPADDRESS:
-            module.fail_json(msg=missing_required_lib('ipaddress'))
+        deps.validate(module, "ipaddress")
+
         try:
-            if 'arpa' not in zone:
+            if 'in-addr.arpa' not in zone:
```

Comment:
That breaks this for IPv6. There, the domain part is `ip6.arpa`. I would simply change this to
```suggestion
            if '.arpa' not in zone:
```
(or not change it at all).

Reply:
reversed